CMakeLists.txt

```diff
@@ -14,14 +14,46 @@ project(
 include(cmake/project-is-top-level.cmake)
 include(cmake/variables.cmake)
 
+find_package(PkgConfig REQUIRED)
+
+find_package(pcre QUIET)
+
+if (NOT pcre_FOUND)
+    # vcpkg provides pcre cmake config as unofficial-pcre
+    find_package(unofficial-pcre CONFIG REQUIRED)
```

Comment:
makes lnav able to use vcpkg pcre package

Reply:
I think the migration to pcre2 is complete, so plain pcre shouldn't be needed...